couchDBSeeder.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const config = require('./config.js');
+
+let url = `http://admin:${config.password}@localhost:5984/product-gallery/_bulk_docs`;
+
+let seedScript = (dbCount = 1) => {
+  let dbCounter = dbCount;
+  let bulk = {
+    docs: []
+  };
+  for (var i = 0; i < 5000; i++) {
+    bulk.docs.push({productId: dbCounter, images: ['https://picsum.photos/200', 'https://picsum.photos/200', 'https://picsum.photos/200']});
+    dbCounter++;
+  }
+  axios.post(url, bulk)
```

Comment:
woah this is tight! so you're seeding batches of 1k records via post req to your couch db?!

Reply:
@HannahManfredi batches of 5k (ignore the console log), script took less than 10min.